source/newcommitter.md

```diff
@@ -29,63 +29,63 @@ The following are some points to consider when assessing a candidate's qualifica
 ### Ability to work cooperatively with peers. 
 How do we evaluate?
 
-  - By the interactions they have through mail.
+  - By the interactions they have through email.
   - By how they respond to criticism.
   - By how they participate in the group decision-making process.
 
 <a name="NewCommitter-Abilitytobeamentor."></a>
 ### Ability to be a mentor.
 How do we evaluate?
 
-  - By the interactions they have through mail.
-  - By how clear they are and how willing they are to point at appropriate background
-materials (or even create them).
+  - By the interactions they have through email.
+  - By how clear they are and how willing they are to identify or even create appropriate background
+materials.
 
 <a name="NewCommitter-Community"></a>
 ### Community
 How do we evaluate?
 
-  - By the interactions they have through mail.
+  - By the interactions they have through email.
   - Do they help to answer questions raised on the mailing list; do they show a helpful 
 attitude and respect for other people's ideas?
 
 <a name="NewCommitter-Committment"></a>
 ### Commitment
 How do we evaluate?
 
-  - By time given to the project already
-  - By sticking through tough issues
-  - By helping on not-so-fun tasks as well
+  - By time already given to the project.
+  - By how well they stick with the process through tough issues.
+  - By how they help on not-so-fun tasks.
 
 <a name="NewCommitter-Personalskill/ability"></a>
 ### Personal skill/ability
 How do we evaluate?
 
   - A solid general understanding of the project. 
-  - Quality of discussion in mail.
-  - Patches (where applicable) easy to apply with only a cursory review.
+  - Quality of discussion in email.
+  - Whether their patches (where applicable) are easy to apply with only a cursory review.
 
 <a name="NewCommitter-NewCommitterProcess"></a>
 ## New Committer Process
 
 This section describes a typical Apache project's process for handling the
-vote to add a new committer. Templates mentioned in the process appear at
+vote to add a new committer. Templates mentioned in the process appear
 later in this document.
 
 <a name="NewCommitter-Summary"></a>
 ### Summary
 
 1. Call a vote (templates/committerVote.txt)
 1. Close the vote
-1. Invite the new committer (templates/committerInvite.txt)
+1. If the result is positive, invite the new committer (templates/committerInvite.txt)
 
-If they accept, then do:
+If they accept, then:
 
 1. Accept the committer (templates/committerAccept.txt)
 1. Wait until we see that receipt of CLA is recorded
 1. Request creation of the committer account (template/committerCreate.txt)
-   1. now wait until root says it is done
-   1. PMC Chair to enable svn and other access
+   1. Wait until root says it is done
```

Comment:
I have additional edits to the onboarding process that will wait until this PR is accepted.